test/jdk/java/awt/print/PrinterJob/PrintTextTest.java

```diff
@@ -21,193 +21,212 @@
-            "the on-screen rendering for that *same* page.",
-            "Some cases may look odd but its intentional. Verify",
-            "it looks the same on screen and on the printer.",
-            "Note that text does not scale linearly from screen to printer",
-            "so some differences are normal and not a bug.",
-            "The easiest way to spot real problems is to check that",
-            "any underlines are the same length as the underlined text",
-            "and that any rotations are the same in each case.",
-            "Note that each on-screen page is printed in both portrait",
-            "and landscape mode",
-            "So for example, Page 1/Portrait, and Page 1/Landscape when",
-            "rotated to view properly, should both match Page 1 on screen.",
-        };
-        Sysout.createDialogWithInstructions(instructions);
-
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.GraphicsEnvironment;
+import java.awt.RenderingHints;
+import java.awt.font.FontRenderContext;
+import java.awt.font.GlyphVector;
+import java.awt.font.TextLayout;
+import java.awt.geom.AffineTransform;
+import java.awt.geom.Point2D;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import java.text.AttributedCharacterIterator;
+import java.text.AttributedString;
+import java.util.HashMap;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JTabbedPane;
+
+public class PrintTextTest {
+
+    static final String INSTRUCTIONS = """
+        This tests that printed text renders similarly to on-screen under a variety
+        of APIs and graphics and font transforms.
+        1. Print to your preferred printer.
+        2. Collect the output.
+        3. Refer to the onscreen buttons to cycle through the on-screen content.
+        4. For each page, confirm that the printed content corresponds to the
+           on-screen rendering for that *same* page. Some cases may look odd but
+           its intentional. Verify it looks the same on screen and on the printer.
+        Note that text does not scale linearly from screen to printer so some
+        differences are normal and not a bug.
+        The easiest way to spot real problems is to check that any underlines are
+        the same length as the underlined text and that any rotations are the same
+        in each case.
+        Note that each on-screen page is printed in both portrait and landscape mode.
+        So for example, Page 1/Portrait, and Page 1/Landscape when rotated to view
+        properly, should both match Page 1 on screen.
+        """;
+
+    public static void main(String[] args) throws Exception {
 
```

Comment:
I think blank lines at the start of methods are redundant. I don't insist on removing them though.